### DISCUSSION TITLE
Why isn't the forms plugin styling my forms?

### DISCUSSION THREAD

**Discussion by ondrejmirtes** [tailwindlabs/tailwindcss#5943] [Help]
Hi,
I'd really appreciate your help in this matter.

This is my website: https://phpstan.org/

This is the source code: https://github.com/phpstan/phpstan/tree/master/website

Because I have:

```js
	plugins: [
		require('@tailwindcss/forms'),
	],
```

in my TailwindCSS config, I'd expect my `<select>` on the homepage to be rounded, but it is not. I succesfully enabled this forms plugin in other places, but I'm yet without luck in this case.

Again - any help is appreciated, I ❤️ TailwindCSS. I wouldn't be able to create any web app without it!

**Comment by NovusOpus** [comment]
If you read the Tailwindcss/forms [Basic Usage Section](https://github.com/tailwindlabs/tailwindcss-forms#basic-usage) you will notice this:

> All of the basic form elements you use will now have some simple default styles that are easy to override with utilities.

and this: 

> Every element has been normalized/reset to a simple visually consistent style that is easy to customize with utilities, even elements like `<select>` or `<input type="checkbox">` that normally need to be reset with `appearance: none` and customized using custom CSS:
> 
> ```HTML
> 
> <select class="px-4 py-3 rounded-full">
>   
> </select>
> 
> 
> <input type="checkbox" class="rounded text-pink-500" />
> ```
You can make your inputs look the way you want, by simple adding Tailwind Classes, like for example(Using some code from your actual site):
```HTML
<div class="flex w-full h-screen items-center justify-center bg-indigo-900">
  <div class="flex items-center justify-center mt-4 flex-col md:flex-row bg-white w-full h-16">
  <!-- ko if: legacyResult() === null -->
  <!-- ko if: !isSharing() -->
	<a href="#" data-bind="click: share" class="inline-flex flex-grow md:flex-grow-0 mx-4 md:mx-0 items-center px-2.5 py-3 rounded-lg text-md font-medium leading-4 bg-gray-100 hover:bg-gray-200 w-auto md:w-32 h-10 justify-center border-gray-300 border">
	<svg fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" class="w-6 h-6"><path d="M8.684 13.342C8.886 12.938 9 12.482 9 12c0-.482-.114-.938-.316-1.342m0 2.684a3 3 0 110-2.684m0 2.684l6.632 3.316m-6.632-6l6.632-3.316m0 0a3 3 0 105.367-2.684 3 3 0 00-5.367 2.684zm0 9.316a3 3 0 105.368 2.684 3 3 0 00-5.368-2.684z"></path></svg>
	<span class="ml-2" data-bind="text: shareText"></span>
	</a>
	<!-- /ko -->

       <!-- /ko -->
  <select class="block mt-4 md:mt-0 mx-4 md:mx-0 md:ml-6 rounded-lg border border-gray-300" data-bind="value: level">
	<option value="0">Level 0</option>
	<option value="1">Level 1</option>
	<option value="2">Level 2</option>
	<option value="3">Level 3</option>
	<option value="4">Level 4</option>
	<option value="5">Level 5</option>
	<option value="6">Level 6</option>
	<option value="7">Level 7</option>
	<option value="8">Level 8</option>
	<option value="9">Level 9</option>
</select>

<label class="inline-flex items-center px-4 mt-4 md:mt-0 ml-0 md:ml-4">
  <input type="checkbox" data-bind="checked: strictRules" class="rounded-md">
  <span class="ml-2">Strict rules</span>
</label>

<label class="inline-flex items-center px-4 mt-4 md:mt-0">
	<input type="checkbox" data-bind="checked: bleedingEdge" class="rounded-md">
		<span class="ml-2">Bleeding edge</span>
</label>
</div>
</div>
```
here's a [TailwindPlay Link](https://play.tailwindcss.com/O38SsG4nSG), so you can see it in action.

**Reply by ondrejmirtes** [reply]
Thanks. But I thought that the select etc. is supposed to be round by default, isn't it?

**Reply by NovusOpus** [reply]
@ondrejmirtes Nope. the default is square like you can see on this [demo](https://tailwindcss-forms.vercel.app/) .

Like it says:

> Every element has been normalized/reset to a simple visually consistent style that is easy to customize with utilities.

The whole point of the @tailwindcss/forms plugin is to make it easy for you to use Tailwind Utility Classes or your Own Custom Utility classes, That means you have full control of how the inputs look.
